### ISSUE TITLE
Not able to create demo project on windows

### ISSUE THREAD

**Issue by PrithvirajChauhan** [facebook/react-native#15422]
> C:\Users\it7>react-native init AwesomeProject
prompt: Directory AwesomeProject already exists. Continue?:  (no) yes
This will walk you through creating a new React Native project in C:\Users\it7\AwesomeProject
Installing react-native...
Consider installing yarn to make this faster: https://yarnpkg.com
{ Error: spawnSync C:\WINDOWS\system32\cmd.exe;C:\Users\it7\AppData\Local\Android\sdk ENOENT
    at exports._errnoException (util.js:1022:11)
    at spawnSync (child_process.js:461:20)
    at execSync (child_process.js:517:13)
    at run (C:\Users\it7\AppData\Roaming\npm\node_modules\react-native-cli\index.js:294:5)
    at createProject (C:\Users\it7\AppData\Roaming\npm\node_modules\react-native-cli\index.js:249:3)
    at C:\Users\it7\AppData\Roaming\npm\node_modules\react-native-cli\index.js:217:7
    at C:\Users\it7\AppData\Roaming\npm\node_modules\react-native-cli\node_modules\prompt\lib\prompt.js:316:32
    at C:\Users\it7\AppData\Roaming\npm\node_modules\react-native-cli\node_modules\async\lib\async.js:142:25
    at assembler (C:\Users\it7\AppData\Roaming\npm\node_modules\react-native-cli\node_modules\prompt\lib\prompt.js:313:9)
    at C:\Users\it7\AppData\Roaming\npm\node_modules\react-native-cli\node_modules\prompt\lib\prompt.js:322:32
  code: 'ENOENT',
  errno: 'ENOENT',
  syscall: 'spawnSync C:\\WINDOWS\\system32\\cmd.exe;C:\\Users\\it7\\AppData\\Local\\Android\\sdk',
  path: 'C:\\WINDOWS\\system32\\cmd.exe;C:\\Users\\it7\\AppData\\Local\\Android\\sdk',
  spawnargs: [ '/s', '/c', '"npm install --save --save-exact react-native"' ],
  cmd: 'npm install --save --save-exact react-native',
  file: 'C:\\WINDOWS\\system32\\cmd.exe;C:\\Users\\it7\\AppData\\Local\\Android\\sdk',
  args:
   [ 'C:\\WINDOWS\\system32\\cmd.exe;C:\\Users\\it7\\AppData\\Local\\Android\\sdk',
     '/s',
     '/c',
     '"npm install --save --save-exact react-native"' ],
  options:
   { stdio: [ [Object], [Object], [Object] ],
     shell: true,
     windowsVerbatimArguments: true,
     file: 'C:\\WINDOWS\\system32\\cmd.exe;C:\\Users\\it7\\AppData\\Local\\Android\\sdk',
     args:
      [ 'C:\\WINDOWS\\system32\\cmd.exe;C:\\Users\\it7\\AppData\\Local\\Android\\sdk',
        '/s',
        '/c',
        '"npm install --save --save-exact react-native"' ],
     envPairs:
      [ 'ALLUSERSPROFILE=C:\\ProgramData',
        'ANDROID_SDK_HOME=C:\\Users\\it7\\AppData\\Local\\Android\\Sdk',
        'APPDATA=C:\\Users\\it7\\AppData\\Roaming',
        'CommonProgramFiles=C:\\Program Files\\Common Files',
        'CommonProgramFiles(x86)=C:\\Program Files (x86)\\Common Files',
        'CommonProgramW6432=C:\\Program Files\\Common Files',
        'COMPUTERNAME=PRUTHVI',
        'ComSpec=C:\\WINDOWS\\system32\\cmd.exe;C:\\Users\\it7\\AppData\\Local\\Android\\sdk',
        'DEFLOGDIR=C:\\ProgramData\\McAfee\\DesktopProtection',
        'FP_NO_HOST_CHECK=NO',
        'HOMEDRIVE=C:',
        'HOMEPATH=\\Users\\it7',
        'LOCALAPPDATA=C:\\Users\\it7\\AppData\\Local',
        'LOGONSERVER=\\\\DOMAIN',
        'NODEJS=C:\\Program Files\\Nodejs',
        'NUMBER_OF_PROCESSORS=4',
        'OneDrive=C:\\Users\\it7\\OneDrive',
        'OS=Windows_NT',
        'Path=C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;D:\\Program Files\\TortoiseSVN\\bin;C:\\Program Files\\Java\\jdk1.7.0_79\\bin;c:\\Program Files (x86)\\Microsoft SQL Server\\110\\Tools\\Binn\\ManagementStudio\\;c:\\Program Files (x86)\\Microsoft SQL Server\\110\\Tools\\Binn\\;c:\\Program Files\\Microsoft SQL Server\\110\\Tools\\Binn\\;c:\\Program Files (x86)\\Microsoft SQL Server\\110\\DTS\\Binn\\;C:\\Users\\it7\\AppData\\Local\\Android\\sdk\\platform-tools;C:\\Program Files\\nodejs\\;C:\\Program Files\\nodejs\\;C:\\Program Files\\Nodejs;C:\\Program Files (x86)\\Skype\\Phone\\;E:\\Softwares\\Jmeter\\apache-jmeter-2.13\\apache-jmeter-2.13\\bin;C:\\Program Files\\Git\\cmd;C:\\Users\\it7\\AppData\\Roaming\\npm;C:\\Program Files\\nodejs\\;E:\\Softwares\\Jmeter\\apache-jmeter-2.13\\apache-jmeter-2.13\\bin',
        'PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JSE;.WSF;.WSH;.MSC',
        'PROCESSOR_ARCHITECTURE=AMD64',
        'PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 60 Stepping 3, GenuineIntel',
        'PROCESSOR_LEVEL=6',
        'PROCESSOR_REVISION=3c03',
        'ProgramData=C:\\ProgramData',
        'ProgramFiles=C:\\Program Files',
        'ProgramFiles(x86)=C:\\Program Files (x86)',
        'ProgramW6432=C:\\Program Files',
        'PROMPT=$P$G',
        'PSModulePath=C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\;c:\\Program Files (x86)\\Microsoft SQL Server\\110\\Tools\\PowerShell\\Modules\\',
        'PUBLIC=C:\\Users\\Public',
        'SESSIONNAME=Console',
        'SystemDrive=C:',
        'SystemRoot=C:\\WINDOWS',
        'TEMP=C:\\Users\\it7\\AppData\\Local\\Temp',
        'TMP=C:\\Users\\it7\\AppData\\Local\\Temp',
        'USERDNSDOMAIN=F2F.COM',
        'USERDOMAIN=F2F',
        'USERDOMAIN_ROAMINGPROFILE=F2F',
        'USERNAME=it7',
        'USERPROFILE=C:\\Users\\it7',
        'VBOX_MSI_INSTALL_PATH=C:\\Program Files\\Oracle\\VirtualBox\\',
        'VS90COMNTOOLS=D:\\Program Files (x86)\\Microsoft Visual Studio 9.0\\Common7\\Tools\\',
        'VSEDEFLOGDIR=C:\\ProgramData\\McAfee\\DesktopProtection',
        'windir=C:\\WINDOWS' ] },
  envPairs:
   [ 'ALLUSERSPROFILE=C:\\ProgramData',
     'ANDROID_SDK_HOME=C:\\Users\\it7\\AppData\\Local\\Android\\Sdk',
     'APPDATA=C:\\Users\\it7\\AppData\\Roaming',
     'CommonProgramFiles=C:\\Program Files\\Common Files',
     'CommonProgramFiles(x86)=C:\\Program Files (x86)\\Common Files',
     'CommonProgramW6432=C:\\Program Files\\Common Files',
     'COMPUTERNAME=PRUTHVI',
     'ComSpec=C:\\WINDOWS\\system32\\cmd.exe;C:\\Users\\it7\\AppData\\Local\\Android\\sdk',
     'DEFLOGDIR=C:\\ProgramData\\McAfee\\DesktopProtection',
     'FP_NO_HOST_CHECK=NO',
     'HOMEDRIVE=C:',
     'HOMEPATH=\\Users\\it7',
     'LOCALAPPDATA=C:\\Users\\it7\\AppData\\Local',
     'LOGONSERVER=\\\\DOMAIN',
     'NODEJS=C:\\Program Files\\Nodejs',
     'NUMBER_OF_PROCESSORS=4',
     'OneDrive=C:\\Users\\it7\\OneDrive',
     'OS=Windows_NT',
     'Path=C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;D:\\Program Files\\TortoiseSVN\\bin;C:\\Program Files\\Java\\jdk1.7.0_79\\bin;c:\\Program Files (x86)\\Microsoft SQL Server\\110\\Tools\\Binn\\ManagementStudio\\;c:\\Program Files (x86)\\Microsoft SQL Server\\110\\Tools\\Binn\\;c:\\Program Files\\Microsoft SQL Server\\110\\Tools\\Binn\\;c:\\Program Files (x86)\\Microsoft SQL Server\\110\\DTS\\Binn\\;C:\\Users\\it7\\AppData\\Local\\Android\\sdk\\platform-tools;C:\\Program Files\\nodejs\\;C:\\Program Files\\nodejs\\;C:\\Program Files\\Nodejs;C:\\Program Files (x86)\\Skype\\Phone\\;E:\\Softwares\\Jmeter\\apache-jmeter-2.13\\apache-jmeter-2.13\\bin;C:\\Program Files\\Git\\cmd;C:\\Users\\it7\\AppData\\Roaming\\npm;C:\\Program Files\\nodejs\\;E:\\Softwares\\Jmeter\\apache-jmeter-2.13\\apache-jmeter-2.13\\bin',
     'PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JSE;.WSF;.WSH;.MSC',
     'PROCESSOR_ARCHITECTURE=AMD64',
     'PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 60 Stepping 3, GenuineIntel',
     'PROCESSOR_LEVEL=6',
     'PROCESSOR_REVISION=3c03',
     'ProgramData=C:\\ProgramData',
     'ProgramFiles=C:\\Program Files',
     'ProgramFiles(x86)=C:\\Program Files (x86)',
     'ProgramW6432=C:\\Program Files',
     'PROMPT=$P$G',
     'PSModulePath=C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules\\;c:\\Program Files (x86)\\Microsoft SQL Server\\110\\Tools\\PowerShell\\Modules\\',
     'PUBLIC=C:\\Users\\Public',
     'SESSIONNAME=Console',
     'SystemDrive=C:',
     'SystemRoot=C:\\WINDOWS',
     'TEMP=C:\\Users\\it7\\AppData\\Local\\Temp',
     'TMP=C:\\Users\\it7\\AppData\\Local\\Temp',
     'USERDNSDOMAIN=F2F.COM',
     'USERDOMAIN=F2F',
     'USERDOMAIN_ROAMINGPROFILE=F2F',
     'USERNAME=it7',
     'USERPROFILE=C:\\Users\\it7',
     'VBOX_MSI_INSTALL_PATH=C:\\Program Files\\Oracle\\VirtualBox\\',
     'VS90COMNTOOLS=D:\\Program Files (x86)\\Microsoft Visual Studio 9.0\\Common7\\Tools\\',
     'VSEDEFLOGDIR=C:\\ProgramData\\McAfee\\DesktopProtection',
     'windir=C:\\WINDOWS' ],
  stderr: null,
  stdout: null,
  pid: 0,
  output: null,
  signal: null,
  status: null,
  error: null }
Command `npm install --save --save-exact react-native` failed.

**Comment by ashafizullah** [comment]
The project with name AwesomeProject is already exist, try to delete it first, then try to make a project again.

**Comment by PrithvirajChauhan** [comment]
@ashafizullah  Deleted existing existing but still same error.

**Comment by ashafizullah** [comment]
@PrithvirajChauhan Or you can try to make a project in other drive, maybe in D: or E: drive.

**Comment by PrithvirajChauhan** [comment]
@ashafizullah Trying to create project in D: drive but still same error. 

My npm version : 3.10.9
react-native-cli: 2.0.1
react-native: n/a - not inside a React Native project directory

**Comment by hramos** [comment]
Hey, thanks for reporting this issue!

It looks like your description is missing some necessary information, or the list of reproduction steps is not complete. Can you please add all the details specified in the [template](https://github.com/facebook/react-native/blob/master/.github/ISSUE_TEMPLATE.md)? This is necessary for people to be able to understand and reproduce the issue being reported.

I am going to close this, but feel free to open a new issue with the additional information provided. Thanks!

**Comment by PrithvirajChauhan** [comment]
So i am new to react-native development. I have followed below link for setup in my windows 10 environment. 
I have installed 
Step 1 : Install Required Apps
Install following applications
nodeJS
Python
I have installed React native using this comman 
`npm install -g react-native-cli`

Now to Create project i am using below command 
`react-native init AwesomeProject`  

but getting error log i have commented above.
